lib/register.php

```diff
@@ -286,6 +286,18 @@ function gutenberg_post_has_blocks( $post_id ) {
 }
 
 /**
+ * Determine whether a content string contains blocks.
+ *
+ * @since 1.3.0
+ *
+ * @param string $content Content to test.
+ * @return bool Whether the content contains blocks.
+ */
+function content_has_blocks( $content ) {
```

Comment:
If we're introducing this function, I think we should refactor `gutenberg_post_has_blocks` to use it as well, rather than repeat the `strpos` implementation.

Reply:
Good catch, updated `gutenberg_post_has_blocks` to use the new function